### PR TITLE
Updated new quickstarts titles and description.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket URL's:** 
1. https://issues.newrelic.com/browse/NR-31187
2. https://issues.newrelic.com/browse/NR-31202
3. https://issues.newrelic.com/browse/NR-30929
4. https://issues.newrelic.com/browse/NR-31209

**Description:** 
Updated new quickstarts (Ansible tower, C# and Typescript) titles and descriptions in the metadata file.

### CODE REVIEW

File: src/data/quickstart-metadata.json [hunk 1]
```diff
@@ -106,5 +106,17 @@
     "wordpress": {
         "title": "WordPress Performance Monitoring | New Relic",
         "description": "WordPress monitoring tool by New Relic helps business users and developers monitor WordPress website performance with New Relic. Install Now!"
+    },
+    "typescript": {
+        "title": "Typescript Application Performance Monitoring | New Relic",
+        "description": "Typescript monitoring tool by New Relic helps business users and developers monitor Typescript performance."
+    },
+    "csharp": {
```
Comment (pkosanam) [inline src/data/quickstart-metadata.json:114]: please change the slug name to "c-sharp"

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: src/data/quickstart-metadata.json [hunk 2]
```diff
@@ -1,110 +1,122 @@
...
-    },
-    "tomcat": {
-        "title": "Apache Tomcat Performance Monitoring | New Relic",
-        "description": "Apache Tomcat monitoring tool by New Relic helps business users and developers monitor Tomcat web server performance with New Relic. Install Now!"
-    },
-    "wordpress": {
-        "title": "WordPress Performance Monitoring | New Relic",
-        "description": "WordPress monitoring tool by New Relic helps business users and developers monitor WordPress website performance with New Relic. Install Now!"
-    }
+  "python": {
```
Comment (pkosanam) [inline src/data/quickstart-metadata.json:2]: @pkudikyala, please check your line indentation

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: src/data/quickstart-metadata.json (hunk 2 above)
Comment (pkudikyala) [reply src/data/quickstart-metadata.json:2]: Resolved line indentation and pushed

**Review by aswanson-nr — APPROVED** [review]

**Review by zstix — APPROVED** [review]

**Comment by aswanson-nr** [comment]
Hey @pkosanam, it just needs you to approve the changes then github will let us merge!

**Pull request merged**